src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  background-color: black;
+  margin: 0;
+}
+
+.container--blue {
+  background-color: #308dc0;
+  width: 328px;
+  padding: 10px;
+  margin: 20px;
+  margin-bottom: 40px;
+  border: 1px solid red;
+}
+
+.container--yellow {
+  background-color: #ffa500;
+  padding: 10px;
+  border: 1px solid red;
+  margin: 0;
+}
+
+.container--lightblue {
+  padding: 10px;
+  margin: 0;
+  background-color: #ddd;
+  border: 1px solid red;
+}
```

Comment:
The theme of the site may be changed and binding class names to colors is bad practice. You can think of these blocks as content, wrapper, and inner wrapper. Example: outer, inner, wrapper, content, etc.